core/templates/pages/exploration-editor-page/feedback-tab/templates/create-feedback-thread-modal.component.html

```diff
@@ -3,13 +3,13 @@ <h3>Start New Feedback Thread</h3>
 </div>
 
 <div class="modal-body">
-  <p>Thread subject: <input class="oppia-feedback-modal-input oppia-autofocus" type="text" ng-model="newThreadSubject" autofocus></p>
-  <p>Message text <textarea class="oppia-feedback-modal-textarea" ng-model="newThreadText" rows="6"></textarea></p>
+  <p>Thread subject: <input class="oppia-feedback-modal-input oppia-autofocus" type="text" [(ngModel)]="newThreadSubject" autofocus></p>
+  <p>Message text <textarea class="oppia-feedback-modal-textarea" [(ngModel)]="newThreadText" rows="6"></textarea></p>
 </div>
 
 <div class="modal-footer">
-  <button class="btn btn-secondary" ng-click="cancel()">Cancel</button>
-  <button class="btn btn-success" ng-click="create(newThreadSubject, newThreadText)" ng-disabled="!newThreadSubject || !newThreadText">Create Thread</button>
+  <button class="btn btn-secondary" (click)="cancel()">Cancel</button>
+  <button class="btn btn-success" (click)="create(newThreadSubject, newThreadText)" ng-disabled="!newThreadSubject || !newThreadText">Create Thread</button>
```

Comment:
does ng-disabled work? I think it needs to be handled differently post migration

Reply:
no, it's not working earlier, I have mistakenly left this unchanged, fixed now

Reply:
Done